testsuite/Makefile

```diff
@@ -95,7 +97,11 @@ parallel-%: lib tools
 	     "$(MAKE) $(NO_PRINT) exec-one DIR={} 2>&1" \
 	 | tee _log
 	@$(MAKE) $(NO_PRINT) retries
-	@$(MAKE) report
+	@if [ '$*' = '*' ] ; then \
+           $(MAKE) RAN_ALL=1 report ; \
+         else \
+           $(MAKE) report ; \
+         fi
```

Comment:
I quickly tested this on Ubuntu with GNU parallel and appear not to have broken it...